src/main/java/com/wellsky/workflow/WorkflowApplication.java

```diff
@@ -6,7 +6,14 @@
 @SpringBootApplication
 public class WorkflowApplication {
 
+    public String PASSWORD = "SECRETPASSWORD";
+
     public static void main(String[] args) {
+        //Bad variable name
+        String SuperHorrible_VarName="";
+        String BAD="";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://hhh-aks.eastus.cloudapp.azure.com/coding_rules#rule_key=squid%3AS00117)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "BAD" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://hhh-aks.eastus.cloudapp.azure.com/coding_rules#rule_key=squid%3AS1481)